packages/plugins/trpc/package.json

```diff
@@ -40,13 +40,15 @@
         "@trpc/server": "^10.32.0",
         "@types/jest": "^29.5.0",
         "@types/node": "^18.0.0",
-        "@types/prettier": "^2.7.2",
         "@zenstackhq/testtools": "workspace:*",
         "copyfiles": "^2.4.1",
         "jest": "^29.5.0",
         "next": "^13.4.7",
         "rimraf": "^3.0.2",
         "ts-jest": "^29.0.5",
         "typescript": "^4.9.4"
+    },
+    "optionalDependencies": {
+        "@types/prettier": "^2.7.2"
```

Comment:
prettier v3 exports ts types so for v3 you don't want to clobber those ts namespaces with the introduction of `@types/prettier`